utilities/pipelines/staticValidation/compliance/module.tests.ps1

```diff
@@ -34,6 +34,10 @@ BeforeDiscovery {
     # Import any helper function used in this test script
     Import-Module (Join-Path $PSScriptRoot 'helper' 'helper.psm1') -Force
 
+    # Load used functions
+    . (Join-Path $repoRootPath 'utilities' 'pipelines' 'publish' 'helper' 'Get-ModuleTargetVersion.ps1')
+    . (Join-Path $repoRootPath 'utilities' 'pipelines' 'publish' 'helper' 'Get-ModulePublishedVersions.ps1')
```

Comment:
Should be added to the `helper.psm1` file. With them loaded in there (and given that it's imported as a module) you'll have the functions also available in the test scope and don't need to load them in over and over.